dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/HTTPConduit.java

```diff
@@ -303,6 +303,24 @@ public abstract class HTTPConduit
 
     private volatile boolean clientSidePolicyCalced;
 
+    // Liberty Change begin
+    private static boolean enableAutoRedirect;
```

Comment:
Do we want this function to be dynamic?

Reply:
No, it will cause performance issues if we have to check the JVM property for every WS request.